src/wp-includes/update.php

```diff
@@ -128,15 +128,20 @@ function wp_version_check( $extra_stats = array(), $force_check = false ) {
 	if ( is_array( $extra_stats ) )
 		$post_body = array_merge( $post_body, $extra_stats );
 
-	$url = $http_url = 'http://api.wordpress.org/core/version-check/1.7/?' . http_build_query( $query, null, '&' );
+	/**
+	 * The auto-update API URL needs to be changed to point to the ClassicPress API instead of WP
+	 *
+	 * @since CP-1.0.0
+	 */
```

Comment:
Let's leave this comment out.  The change, and the reason for it, is fairly obvious, and the `git` commit history will point back to this PR for more information.